Plugins/SequencePlugin/Source/SequencePlugin/Public/Types/ERC20.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "UObject/Object.h"
+#include "Sequence/Transactions.h"
+#include "ERC20.generated.h"
+
+UCLASS(BlueprintType, Blueprintable)
+class SEQUENCEPLUGIN_API UERC20 : public UObject
```

Comment:
Should probably add the Approve function

Reply:
added